src/stan/io/stan_csv_reader.hpp

```diff
@@ -256,14 +256,14 @@ namespace stan {
             break;
 
           if (comment_line) {
-            if (line.find("(Warm-up)") != std::string::npos) {
-              int left = 17;
-              int right = line.find(" seconds");
+            if (line.find("Warmup") != std::string::npos) {
```

Comment:
@betanalpha, can we split this into its own pull request so it can patch `develop` now? This seems useful.